models/models_methods_test.go

```diff
@@ -791,91 +838,282 @@ func TestValidateConfig(t *testing.T) {
 		require.Equal(t, 0, len(warnings))
 	}
 
-	t.Log("Invalid bitriseData ID - empty")
+	t.Log("Invalid bitriseData - pipeline ID empty")
 	{
 		bitriseData := BitriseDataModel{
 			FormatVersion: "1.4.0",
-			Workflows: map[string]WorkflowModel{
-				"": WorkflowModel{},
+			Pipelines: map[string]PipelineModel{
+				"": PipelineModel{},
 			},
 		}
 		warnings, err := bitriseData.Validate()
-		require.EqualError(t, err, "invalid workflow ID (): empty")
+		require.EqualError(t, err, "invalid pipeline ID (): empty")
 		require.Equal(t, 0, len(warnings))
 	}
 
-	t.Log("Invalid bitriseData ID - contains: `/`")
-	{
+	for _, specConfig := range []struct {
```

Comment:
It would be enough to test it with only 1 invalid character, no need for table testing.